CS3219-AY2425S1/cs3219-ay2425s1-project-g02
# [Do not merge] Frontend: Create account page

## changes
- add createAccountPage
- linked from loginPage
- validation
  - required fields
  - type=email for regex email format check
  - type=password to hide password and show dots
- successfully creates an account on firebase
- accompanying success/fail alert messages
